taxi/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.contrib import admin
+from django.urls import path
+from taxi.views import homeland
+from taxi.views import index
+
+
+
+urlpatterns = [
+    path("homeland/", homeland),
+    path("", index)

```

Comment:
you should add name to index

Reply:
added

Reply:
name="index" will simplify work in futere